pkgs/by-name/op/openmpi/package.nix

```diff
@@ -47,6 +48,14 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-vUGD/LxDR3wlR5m0Kd8abldsBC50otL4s31Tey/5gVc=";
   };
 
+  # This patch can be removed with the next openmpi update (>5.0.6)
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/open-mpi/ompi/pull/13003.patch";
```

Comment:
The best link to put for such merged PRs, is the link of the merge commit. IIRC that this link may die or change if the author of the PR will delete or update the branch they used for the PR.